maas-schemas/schemas/core/components/common.json

```diff
@@ -71,6 +71,15 @@
       "type": "string",
       "enum": ["whim", "wechat"]
     },
+    "requestId": {
+      "oneOf": [
+        { "$ref": "http://maasglobal.com/core/components/units.json#/definitions/uuid" },
+        {
+          "type": "string",
+          "enum": ["UNKNOWN"]
```

Comment:
If not available, isn't it made more sense just being null?

Reply:
Maybe :D

Reply:
In this way there is a clear indication that the routes query was executed without a `requestId`, and is soemthing positive to base other things on, e.g.  searching logs for bad requests to omit them from further use.

Reply:
I'd still suggest using null. Null is very clear, comparing to what you think about "undefined"